Source/Core/Row.swift

```diff
@@ -63,6 +63,13 @@ open class RowOf<T>: BaseRow where T: Equatable{
             return _value
         }
     }
+    
+    /// The default value of this row. Sets the value property when setted or when the setDefaultValue is called.
+    open var defaultValue: T? {
+        didSet {
+            value = defaultValue
```

Comment:
I would prefer to avoid this value property set up when configuring `defaultValue`. Otherwise we would have some unexpected behavior. 
Please remove `didSet` and update the test accordingly. 
I guess we can also rename the property to `resetValue` to avoid any misinterpretation.

Reply:
The main idea was setting the default value / value once in the code. I mean, if I set the default value, that is the initial value for the row and if I don't want to use a default value then I can still set it through the value property. Do you really think it would lead to an unexpected behavior?

Reply:
I guess we would run into setting the default value somewhere after setting the row's value and in this particular case in most of the time we will want to maintain the row `value` value. 

I guess the default value is super useful for reseting the row or the entire form by calling `form.reset()` that would be implemented.

Reply:
I understand. I have made the changes you asked. I renamed the method to `resetRowValue`, because it cannot have the same name of the property.
Do you have any idea on how to implement the `form.reset()` ?